server/controllers/tickets.js

```diff
@@ -9,7 +9,7 @@ module.exports.getPanelTickets = (req, res) => {
   }
   //req.query.panel_id used to be req.params.panel_id but axios GET can only put in req.query
   var panelId = req.query.panel_id;
-  dbhelper.getTicketsByPanel(parseInt(panelId))
+  dbhelper.getTicketsByPanel(panelId)
```

Comment:
Without converting to integer with parseInt(panelId), there's a chance that dbhelper will confuse the argument as a string and fail. I've had those failures before which is why I do parseInt on anything that is supposed to be an integer when passing into a dbhelper function.